### ISSUE TITLE
Add option to save posted videos to device gallery

### ISSUE THREAD

**Issue by mtwzim** [bluesky-social/social-app#5310]
**Is your feature request related to a problem? Please describe.**

It's a feature request.

Bluesky recently introduced the ability to post videos, but users cannot save these videos directly to their device's gallery. Other apps already provide this functionality, and it would be useful for Bluesky to offer the same, ensuring a consistent user experience.

**Describe the solution you'd like**

I would like Bluesky to implement a feature that allows users to save posted videos directly to their device’s gallery, just like they can with photos. This would make it easier to access and share videos outside the app.

This feature would complement the recently introduced ability to post videos, enhancing the overall user experience by providing additional functionality. Allowing users to save videos directly to their device would offer greater flexibility in managing and sharing content, aligning Bluesky with the capabilities of other apps and improving overall user satisfaction.

**Additional context**

Existing 'Save' Button for Photos:
![WhatsApp_Image_2024-09-12_at_19 13 58_1_42](https://github.com/user-attachments/assets/4c9488a3-81f7-41e3-8bbd-815cdc373dd1)

**Comment by surfdude29** [comment]
I agree that this would be useful. I think I remember seeing somewhere that the team had considered this, but decided to hold off on it for the initial release while they thought about it some more.

In the meantime, cobalt.tools ([website](https://cobalt.tools/), [on Bluesky](https://bsky.app/profile/cobalt.tools)) has added support for saving Bluesky videos (h/t @mozzius).
